### ISSUE TITLE
Changelog 3.6

### ISSUE THREAD

**Issue by Bibi56** [pbatard/rufus#1349]
Issue description
-----------------
In https://rufus.ie/ you mention:

    Version 3.6 (2019.07.18)

    Add support for persistent partitions [EXPERIMENTAL]
    (Note: The above won't work with Ubuntu until Ubuntu bug #1489855 is properly fixed)

According to https://bugs.launchpad.net/ubuntu/+source/casper/+bug/1489855/comments/35

    This bug was fixed in the package casper - 1.414

So maybe you can mention that it's solved on Ubuntu where casper has been upgraded to `1.414` or higher.

Thanks for your work. 

Log
---
None

**Comment by pbatard** [comment]
Yes, I know. If you read the bug report, you'll see that I have been the person replying to Michael and testing his patch.

The problem is that there is no formal release of Ubuntu where the bugfix has been applied. That will not happen before [October 17th](https://wiki.ubuntu.com/EoanErmine/ReleaseSchedule). Which means that there's little point of mentioning a fix that is going to take months to find its way to regular users (and, since I mention the bug explicitly, anybody interested in the issue can easily find the information you mention above).

Finally, there are 2 additional matters to consider:
1. Once something has been released, the ChangeLog for that release is pretty much set in stone. You may fix some typos here and there, but, since the ChangeLog is meant to capture the state of things at the time of release (i.e. it's a snapshot of the conditions that existed at the time of release), and that bug was not properly fixed then, you don't want to alter the text of the ChangeLog, even if things have partially changed. Instead, what you do is mention such changes in the next release entry (which is what I'm planning to do). You may think: _"What's the harm?"_, but I can tell you that there are some people who get very nervous about release notes changing on a whim, because, when that happens, you can never be quite sure that the application itself hasn't changed too, and that creates __TRUST__ issue with the software. So, in order to preserve __TRUST__ with my users, I consider that, once a release has been issued, the ChangeLog is non-alterable save for typos. If a situation described in the ChangeLog evolved, then this evolution will need to be described in a __new__ entry.
2. Because of this very bug, I have __not__ been adding a new feature, which I would have added otherwise, that __actually__ enables persistence with Ubuntu (which boils down to modifying the boot kernel option to add `persistent`). So, even if someone were to use a version of Ubuntu with a fixed casper, it still __won't work out of the box with Rufus__. And this actually creates an issue for me because, now I need to weigh the impact of adding the `persistent` kernel option against the estimate of how many people are going to try to use older (unfixed) versions of Ubuntu where enabling this option will actually prevent boot (which is the reason why I don't alter the kernel options in 3.6). And I also have to weigh in how this will work for other distros, since the bug that was fixed affects upstream casper, rather than just the Ubuntu version, and I know that other distros such as Mint suffer from the exact same problem. So, while one might consider that, if I were to release a version of Rufus on Oct. 17 that adds `persistent` to the kernel options, I'd be mostly in the clear, it's not as clear cut as that, and at the moment, I am actually tempted not to alter anything with regards to persistence before 2020, in order to give a chance for most distros to sort out their casper issues. In which case, I'd rather __not__ float the idea that all that's needed is get your hands on an ISO where casper has been updated post 1.414 for persistence to work with bootable media created by Rufus, when this is most certainly not the case.

In other words, I'm not planning to alter anything from the ChangeLog with regards to what's currently mentioned about the persistence issue, because doing so would actually be more detrimental than helpful. People who are interested in the issue can already find the bug and figure things out for themselves.
